requirements.txt

```diff
@@ -15,3 +15,5 @@ panda-gym==1.1.1 # tmp fix: until compatibility with panda-gym v2
 rliable>=1.0.5
 wandb
 ale-py==0.7.4 # tmp fix: until new SB3 version is released
+# TODO: replace with release
+git+https://github.com/huggingface/huggingface_sb3
```

Comment:
We can now replace with the release :hugs:

Reply:
hmm, the new release still has pickle5 as dependency I think.